app/controllers/health_cards_controller.rb

```diff
@@ -13,7 +13,7 @@ def show
       format.healthcard { render json: exporter.download }
       format.html do
         @jws_encoded_details = exporter.jws
-        @health_card = HealthCards::COVIDHealthCard.from_jws @jws_encoded_details
+        @health_card = HealthCards::COVIDImmunizationCard.from_jws @jws_encoded_details
```

Comment:
Right now, we can only create `COVIDImmunizationCard` classes, and not `COVIDLabResultCard` classes. This seems wrong to not be able to have the `#laboratory` types in VCs that contain only lab data. 

Is this being addressed in [DVCI-87](https://jira.mitre.org/browse/DVCI-87)?

Reply:
Yep it's being addressed in my task!